README.md

```diff
@@ -79,6 +79,10 @@ There is also the community driven PowerShell Slack Team which you can sign up f
 
 [Slack Sign up]: http://slack.poshcode.org
 
+## Addons and libraries
```

Comment:
I believe "Add-Ons" is better term.
Also please add it in https://github.com/PowerShell/PowerShell/blob/master/.spelling